content/release-notes/1.16.0.md

```diff
@@ -0,0 +1,33 @@
+---
+date: 2020-06-01T08:00:00-07:00
+linktitle: "1.16.0"
+title: "1.16.0"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<features>}}
+* Support for HTTP_PROXY and NO_PROXY environment variables.
+* CronJobs now support private images.
+* Gitops cluster tracers
```

Comment:
I don't understand these two above. The first seems customer-impacting (is it a bug?), the latter seems like it's technical and maybe can be removed?

Reply:
The "gitops cluster tracers" should be included as a change, but explained better...  @salahalsaleh will you take a pass at explaining the change here (... added annotations to ...)

Reply:
```suggestion
* Gitops cluster tracers - added the app slug and sequence as annotations to the application pods
```

Reply:
@salahalsaleh how about:

* Added KOTS metadata to all application manifests. A unique identifier for the local application and the version sequence will be injected as annotations to all deployed manifests.